extension/js/common/composer/composer-draft.ts

```diff
@@ -173,38 +173,30 @@ export class ComposerDraft extends ComposerComponent {
     }
   }
 
-  private async decryptAndRenderDraft(encryptedArmoredDraft: string, headers: { from?: string; to: string[] }): Promise<boolean> {
+  private async decryptAndRenderDraft(encryptedArmoredDraft: string, headers: { subject?: string, from?: string; to: string[], cc: string[], bcc: string[] }): Promise<boolean> {
     const passphrase = await this.app.storagePassphraseGet();
     if (typeof passphrase !== 'undefined') {
       const result = await PgpMsg.decrypt({ kisWithPp: await Store.keysGetAllWithPp(this.urlParams.acctEmail), encryptedData: Buf.fromUtfStr(encryptedArmoredDraft) });
       if (result.success) {
+        if (headers.subject) {
```

Comment:
This looks like it should work, but for whatever reason, when tested, subject is not added. Not sure why